docs/managing-wallets.md

```diff
@@ -29,6 +29,7 @@ in the future
 - The identity of CEOs and/or developers is public
 - Avoid address reuse by displaying a new receiving address for each transaction
   in the wallet UI
+- Uses deterministic ECDSA nonces (RFC 6979)
```

Comment:
@crwatkins Bitcoin Core is considering a [change](https://github.com/bitcoin/bitcoin/pull/13666) (also proposed by @achow101) where it'll be specifying additional entropy into RFC6979 in order to always generate signatures with low-R values, saving one byte from signatures that would otherwise have an R value on the top half of the curve.  Is this consistent with your intention for this policy?  (See previously linked issue for a description of how the new formula is still deterministic even if it isn't fully specified by 6979.)

Reply:
>  Is this consistent with your intention for this policy?

Yes.  It is consistent.  It still does all the _good things_ and still allows for verification in closed source hardware wallets, as well as easier verification everywhere else.

I didn't write the original optional criterium so when I moved it to **required** I considered being more explicit about whether RFC6979 was required and opted not to since it wasn't required to meet my goals.   I left it in as a reference because as well as being the best vetted option (all listed wallets implement it) there are still folks working on wallets that aren't very familiar with the concept.